mne/io/meas_info.py

```diff
@@ -175,6 +175,13 @@ def get_montage(self):
         # so use loc[:3] instead
         ch_pos = {ch_names[ii]: chs[ii]['loc'][:3] for ii in picks}
 
+        # fNIRS uses multiple channels for the same sensors
+        # so we use a private function to format these for
+        # dig montage
+        fnirs_picks = pick_types(info, fnirs=True, exclude=[])
+        if len(ch_pos) == len(fnirs_picks):
```

Comment:
can you clarify what this == test? does it mean it only works when you have only fNIRS channels in the instance?

Reply:
Yes, currently this is implemented so that it will only work if there is only fNIRS channels. 

I have not tried multimodal data. Should I a) change the implementation to work with a mix of data types (e.g. EEG, MEG, and fNIRS)? or b) maybe add a warning to say we don't handle a mix of data types with this function yet? 
Something like...
```python
elif (len(fnirs_picks) > 0) & (len(fnirs_picks) < len(ch_pos)):
    raise ValueError("MNE does not yet support getting the montage for a mix of fNIRS"
                                 "and other data types. Please raise a GitHub issue if you"
                                 "require this feature.")
```

Reply:
> maybe add a warning to say we don't handle a mix of data types with this function yet?

It would be great to make it work for multimodal data, but for now raising an error seems safest. Someone can always in theory pick then `.add_channels` to work around it in the meantime

Reply:
+1